styles/gallery.css

```diff
@@ -0,0 +1,51 @@
+body {
+        background-color: lightblue;
+}
+
+p {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 20px;
+        color: black
+}
+
+.nav {
+        color: darkslategrey;
+        text-align: left;
+        background-color: lightblue;
+        display: inline-block;
+        border-style: dotted;
+        border-color:lightseagreen;
+    }
+
+img {
+    width:150px;
+    height: auto;
+}
+
```

Comment:
Throughout the project, there seems to be inconsistent tab-spacing. It'd probably be good to conform to 2 or 4 spaces for tabs.